### ISSUE TITLE
Unable to delete uploaded documents

### ISSUE THREAD

**Issue by AnselmJeong** [Nutlope/pdftochat#4]
I am satisfied with the quality of summaries and other Q&A services more than any other service. 
However, I am concerned that I cannot find a way to delete a document once it's uploaded. Wouldn't that mean documents will just keep accumulating? From the user's perspective, I'm also worried about private information being leaked. 

Since I might not know the way to delete a document, I would appreciate your guidance on how to do it.

**Comment by ajaxbo360** [comment]
@AnselmJeong yeah very good point , they will add it soon ,if you look at the readme section on Github , they say that is a feature to be done .

**Comment by Nutlope** [comment]
@AnselmJeong @ajaxbo360 correct, I will add this in a couple days!

**Comment by psbhatbvbcs** [comment]
Hey @Nutlope and @jacoblee93 ! I came across your amazing project yesterday. And saw this project as a perfect place to use my skills. I have created a Pull request for this particular feature. I have tried matching your coding style as much as possible and also plan to contribute to this project further!
